policy-controller/runtime/Cargo.toml

```diff
@@ -7,11 +7,11 @@ license = "Apache-2.0"
 publish = false
 
 [features]
-default = ["openssl-tls", "openssl-vendored"]
-openssl-tls = ["kube/openssl-tls"]
+default = ["rustls-tls"]
```

Comment:
default changed to be kept in line with the binary crate:

https://github.com/linkerd/linkerd2/blob/1d1d3cd97ebad4a883aeb7ae920261431724b6a0/policy-controller/Cargo.toml#L8-L9